public/app/features/api-keys/ApiKeysForm.tsx

```diff
@@ -75,18 +87,9 @@ export const ApiKeysForm: FC<Props> = ({ show, onClose, onKeyAdded }) => {
               <Input type="text" className="gf-form-input" value={name} placeholder="Name" onChange={onNameChange} />
             </div>
             <div className="gf-form">
-              <span className="gf-form-label">Role</span>
-              <span className="gf-form-select-wrapper">
-                <select className="gf-form-input gf-size-auto" value={role} onChange={onRoleChange}>
-                  {Object.keys(OrgRole).map((role) => {
-                    return (
-                      <option key={role} label={role} value={role}>
-                        {role}
-                      </option>
-                    );
-                  })}
-                </select>
-              </span>
+              <InlineField label="Role">
+                <Select inputId="role-select" value={role} onChange={onRoleChange} options={ROLE_OPTIONS} />
+              </InlineField>
             </div>
             <div className="gf-form max-width-21">
               <InlineFormLabel tooltip={tooltipText}>Time to live</InlineFormLabel>
```

Comment:
```suggestion
        <InlineField tooltip={tooltipText} label={'Time to live'}>
              <Input
                type="text"
                placeholder="1d"
                validationEvents={timeRangeValidationEvents}
                value={secondsToLive}
                onChange={onSecondsToLiveChange}
              />
            </InlineField>
```
In in the Add API key form the alignment of TTL seems quite off
![Screenshot 2021-10-01 at 16 02 44](https://user-images.githubusercontent.com/8878045/135624088-9f6ec5e7-8333-4aaf-ac0d-ed5b0625dd74.png)
Using `InlineField` seems to fix the issue.

Reply:
Nice, thanks.